projects/soft_timers_and_you/src/main.c

```diff
@@ -0,0 +1,38 @@
+#include "interrupt.h"
+#include "log.h"
+#include "soft_timer.h"
+#include "wait.h"
+#define CONST_MILLIS_COUNTER 500
+
+typedef struct {
+  uint8_t counter_a;
+  uint8_t counter_b;
+} Counters;
+
+static void prv_counter(SoftTimerId timer_id, void *context) {
+  Counters *counter = context;
+  counter->counter_a++;
+  if (counter->counter_a % 2 == 0) {
```

Comment:
Please move the `LOG_DEBUG("Counter A...")` outside the conditional so you don't have to repeat it